src/app.ts

```diff
@@ -98,7 +98,7 @@ export default class App {
     const prefix = this._configService.get('BOT_TRIGGER');
     const result = await campaignRepository.find({
       where: { roomId: message.channel.id },
-      relations: ['characters'],
```

Comment:
I think we should move this method && any RPG behavior to its own class and load appropriately here. I can see this bloating up and taking a lot of lines and complex behaviors... We might wanna separate this concern from the get-go

Reply:
I'm working on a refactor of this base app class that will help with this then I'll make it gets applied into this.